cypress/e2e/emulator.cy.js

```diff
@@ -1,56 +1,136 @@
+const getIframeBody = () => {
+  return cy
+    .get('iframe[id="FCL_IFRAME"]')
+    .its('0.contentDocument.body').should('not.be.empty')
+    .then(cy.wrap)
+}
+
 describe('Emulator + dev-wallet tests', () => {
+
+  describe('Scenarios that do not interact with kitty items', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:3001/')
+    })
+
+    it('visits header buttons', () => {
+      // Should be the same as homepage
+      cy.get('[data-cy="header right"]').contains('Store').click()
+      cy.get('[data-cy="home"]').contains('MINT YOUR FIRST KITTY ITEM')
+
+      cy.get('[data-cy="header right"]').contains('Marketplace').click()
+      cy.get('[data-cy="marketplace"]').contains('Marketplace').should('exist')
+
+      // Click back to the homepage
+      cy.get('[data-cy="header left"]').click()
+      cy.get('[data-cy="home"]').contains('MINT YOUR FIRST KITTY ITEM')
+    })
+
+    it('logs in as admin', () => {
+      cy.visit('http://localhost:3001/admin/mint')
+
+      cy.get('[data-cy="btn log in admin"]').click()
+      cy.get("input[placeholder=\"Enter Password\"]").type('KittyItems')
+      cy.get("button[type=\"submit\"]").click()
   
-  const getIframeBody = () => {
-    return cy
-      .get('iframe[id="FCL_IFRAME"]')
-      .its('0.contentDocument.body').should('not.be.empty')
-      // wraps "body" DOM element to allow
-      // chaining more Cypress commands, like ".find(...)"
-      // https://on.cypress.io/wrap
-      .then(cy.wrap)
-  }
-
-  it('Visit main page', () => {
-    cy.visit('http://localhost:3001/')
-  })
+      cy.get('[data-cy="header mint"]').should('exist')
```

Comment:
I think it's better to use dashes instead spaced to separate words in a single-value attributes.
For example "header mint" -> "header-mint"
With spaces there is some ambiguity around whether it's one value or an array of 2 values